src/OrchardCore/OrchardCore/Extensions/ExtensionManager.cs

```diff
@@ -366,10 +372,10 @@ private bool HasDependency(IFeatureInfo f1, IFeatureInfo f2)
             return false;
         }
 
-        private int GetPriority(IFeatureInfo feature)
+        private static int GetPriority(IFeatureInfo feature, IExtensionPriorityStrategy[] extensionPriorityStrategies)
         {
             var sum = 0;
-            foreach (var strategy in _extensionPriorityStrategies)
+            foreach (var strategy in extensionPriorityStrategies)
```

Comment:
these 4 lines could be simplified with Linq.

```
var sum = extensionPriorityStrategies.Sum(strategy => strategy.GetPriority(feature))
```

Reply:
This would bring us a lot of additional allocations for removing 3 lines. I'd rather keep it as-is.

Reply:
I trust you. Why why would this add more allocations?

Reply:
Because the Lambda expression needs a closure for the `feature` variable. And as `GetPriority` gets called a lot, those would add up.

Reply:
E.g. https://sharplab.io/#gist:d051810459d9fe5ba06dbf556292622a